src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categoryList: [],
+};
+
+const categoriesSlice = createSlice(
+  {
+    name: 'category',
+    initialState,
+    reducers: {
+      checkStatus: (state) => { state.categoryList = 'Under Construction'; },
+
+    },
```

Comment:
- [ ] what the previous CR meant is that in order to return the string 'Under construction' a condition should be met, which in this case is that `action.payload` should be 'Under Construction', you can do this by including a ternary operator which might look something like below:
```
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },

```